e2e/next-sandbox/pages/storage/list.tsx

```diff
@@ -52,6 +53,9 @@ function Sandbox() {
   const items = useStorage((root) => root.items);
   const me = useSelf();
   const status = useStatus();
+  const storageStatus = useStorageStatus();
+  const smoothStorageStatus1 = useStorageStatus({ smooth: 1 });
+  const smoothStorageStatus2 = useStorageStatus({ smooth: 2 });
```

Comment:
@CTNicholas Hey! Rather than having a single `{ smooth: true }`, I've added two different implementations that could both work. Would you mind giving these a try and decide which one is preferable?

Reply:
https://github.com/liveblocks/liveblocks/assets/83844/59d0abbc-0ab9-406d-96f6-01f3a6db8702

Reply:
I definitely prefer option 2, for a couple of reasons.

- Smoother. We din't want to create a highly visible flicker.
- Liveblocks synchs so quickly it actually makes me mildly suspicious. The artificial delay is nice.

Reply:
Agreed, I think I'd also prefer (1). Wanted to make sure we try them both out "in the wild" first, before committing to it.

Here is the plan:
- I will remove alternative (2) from this PR
- @CTNicholas We need to decide on the final duration before we hardcode it. It's currently hard-coded to 1 second. Should we keep it, or make it faster? Not slower I think?
- @CTNicholas Do you think allowing configuration of this duration makes sense? I.e. allow both `{ smooth: true }` (1 second), or `{ smooth: 600 }` if you want full control? I don't think I like it that much and would prefer picking a good opinionated default value—at least not until we get a request for it. Do you agree?

Reply:
Removed alternative 2 in c52d99fd9254d2508d20db2dd320e6529873d750 👍